src/web/routes/application/enter-code/enter-code.js

```diff
@@ -0,0 +1,41 @@
+const { TEXT, EMAIL } = require('../common/constants')
+
+const pageContent = ({ translate }) => ({
+  title: translate('enterCode.title'),
+  heading: translate('enterCode.heading'),
+  codeLabel: translate('enterCode.codeLabel'),
+  weJustSentASixDigitCodeTo: translate('enterCode.weJustSentASixDigitCodeTo'),
+  itMayTakeAFewMinutesToArrive: translate('enterCode.itMayTakeAFewMinutesToArrive'),
+  requestNewCode: translate('enterCode.requestNewCode'),
+  ifItDoesNotArrive: translate('enterCode.ifItDoesNotArrive'),
+  buttonText: translate('buttons:continue')
+})
+
+const getEmailOrPhoneNumberForConfirmationCode = (claim) => {
```

Comment:
What does this method have to do with the confirmation code, looks like its just getting the email or phone number given their choice, or is there more to come in this method?

Reply:
That's exactly what it's doing. We need to display that on the page.